package.json

```diff
@@ -33,12 +34,14 @@
   },
   "homepage": "https://github.com/mironov/react-redux-loading-bar",
   "peerDependencies": {
+    "@types/react": ">=0.14.55",
```

Comment:
TypeScript developers will need to depend on `@types/react` (as the definitions I added refer to the React `Component`).

However, in reality, all TS devs will already have that dependency so they can use React, so it should be fine to remove.